setup.py

```diff
@@ -49,6 +49,7 @@ def run(self):
     "colorama>=0.3.9",
     "configobj>=5.0.6",
     "gitpython>3",
+    "dircolors>=0.0.4",
```

Comment:
Could you please check if there is a conda package for this?

Reply:
Thanks for pointing me out, @efiop. I haven't found such package in conda and going to deploy it there. Meanwhile you can take a look on my source code changes

Reply:
I decided not to use library